LabeledIconButton/tests/LabeledIconButton-specs.js

```diff
@@ -0,0 +1,223 @@
+import {mount} from 'enzyme';
+import React from 'react';
+
+import LabeledIconButton from '../LabeledIconButton';
+
+describe('LabeledIconButton Specs', () => {
+	test('should render label when given children', () => {
+		const subject = mount(
+			<LabeledIconButton
+				icon="temperature"
+			>
+				Hello LabeledIconButton
+			</LabeledIconButton>
+		);
+
+		const expected = 'Hello LabeledIconButton';
+		const actual = subject.find('label').text();
+
+		expect(actual).toBe(expected);
+	});
+
+	test('should not render children when no children is passed', () => {
+		const subject = mount(
+			<LabeledIconButton icon="temperature" />
+		);
+
+		const expected = '';
+		const actual = subject.find('label').text();
+
+		expect(actual).toBe(expected);
+	});
+
+	test('should render disabled LabeledIconButton', () => {
+		const subject = mount(
+			<LabeledIconButton
+				disabled
+				icon="temperature"
+			>
+				Hello LabeledIconButton
+			</LabeledIconButton>
+		);
+
+		const expected = true;
+		const actual = subject.find('LabeledIconButton').prop('disabled');
+
+		expect(actual).toBe(expected);
+	});
+
+	test('should render highlighted LabeledIconButton', () => {
+		const subject = mount(
+			<LabeledIconButton
+				highlighted
+				icon="temperature"
+			>
+				Hello LabeledIconButton
+			</LabeledIconButton>
+		);
+
+		const expected = true;
+		const actual = subject.find('LabeledIconButton').prop('highlighted');
```

Comment:
Shouldn't we check `className` rather than `prop`?
Same comment for the rest of the tests.